VimeoUpload/Descriptor System/DescriptorManager.swift

```diff
@@ -70,9 +76,21 @@ open class DescriptorManager: NSObject
     // By passing the delegate into the constructor (as opposed to using a public property)
     // We ensure that early events like "load" can be reported [AH] 11/25/2015
     
-    init?(sessionManager: AFURLSessionManager, name: String, archivePrefix: String?, shouldLoadArchive: Bool, documentsFolderURL: URL, delegate: DescriptorManagerDelegate? = nil)
+    init?(sessionManager: AFURLSessionManager,
+          name: String,
+          archivePrefix: String?,
+          shouldLoadArchive: Bool,
+          documentsFolderURL: URL,
+          delegate: DescriptorManagerDelegate? = nil)
     {
-        guard let archiver = DescriptorManagerArchiver(name: name, archivePrefix: archivePrefix, shouldLoadArchive: shouldLoadArchive, documentsFolderURL: documentsFolderURL) else
+        let migrator = ArchiveMigrator(fileManager: FileManager.default)
+        
+        guard let archiver = DescriptorManagerArchiver(name: name,
+                                                       archivePrefix: archivePrefix,
+                                                       shouldLoadArchive: shouldLoadArchive,
+                                                       documentsFolderURL: documentsFolderURL,
+                                                       migrator: migrator)
```

Comment:
@van does `DescriptorManager` need to create the migrator if it's only the archiver who needs it?

Reply:
We spoke IRL, we'll go ahead an instantiate this at use site 👍